components/stacks-network/src/chains_coordinator.rs

```diff
@@ -105,23 +106,24 @@ impl DevnetEventObserverConfig {
         .expect("unable to load network manifest");
 
         let event_observer_config = EventObserverConfig {
-            normalization_enabled: true,
-            grpc_server_enabled: false,
             hooks_enabled: true,
             bitcoin_rpc_proxy_enabled: true,
             event_handlers: vec![],
             chainhook_config: Some(chainhooks),
             ingestion_port: devnet_config.orchestrator_ingestion_port,
             control_port: devnet_config.orchestrator_control_port,
-            bitcoin_node_username: devnet_config.bitcoin_node_username.clone(),
-            bitcoin_node_password: devnet_config.bitcoin_node_password.clone(),
-            bitcoin_node_rpc_url: format!("http://{}", services_map_hosts.bitcoin_node_host),
+            control_api_enabled: true,
+            bitcoind_rpc_username: devnet_config.bitcoin_node_username.clone(),
+            bitcoind_rpc_password: devnet_config.bitcoin_node_password.clone(),
+            bitcoind_rpc_url: format!("http://{}", services_map_hosts.bitcoin_node_host),
+            bitcoin_block_signaling: BitcoinBlockSignaling::Stacks("http://0.0.0.0:20443".into()),
             stacks_node_rpc_url: format!("http://{}", services_map_hosts.stacks_node_host),
             operators: HashSet::new(),
             display_logs: true,
             cache_path: devnet_config.working_dir.to_string(),
             bitcoin_network: BitcoinNetwork::Regtest,
             stacks_network: chainhook_event_observer::chainhook_types::StacksNetwork::Devnet,
+            ordinals_enabled: false,
```

Comment:
@lgalabru 
Could you please double-check here the default values I added? (including bitcoind_* and bitcoin_block_signaling)
Thx